web/src/pages/index.module.css

```diff
@@ -33,7 +33,7 @@
     padding-left: 40px;
   }
 
-  .vastLogo {
+  .tenzirLogo {
     width: 400px;
```

Comment:
@dit7ya could you check out this branch and take a look at the homepage? The new logo is substantially narrower. The `width` parameter doesn't seem to have an effect and I don't know why.

Reply:
This is obsolete now that we removed the landing page altogether.